packages/swingset-liveslots/src/vatDataTypes.d.ts

```diff
@@ -38,9 +39,11 @@ type OmitFirstArg<F> = F extends (x: any, ...args: infer P) => infer R
   ? (...args: P) => R
   : never;
 
-export type KindFacet<O> = RemotableObject & {
+type Remotish<T> = T & RemotableObject & RemotableBrand<{}, T>;
```

Comment:
what makes this "ish". Please document why this type is needed and when to use it.

Reply:
Renamed to `PrimaryRemotable<T>` and explicitly marked as internal just to DRY up the typing of `KindFacet<O>`.